<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <group ns="high_level">
    <rosparam file="$(find gripper_controls)/config/high_lvl_params.yaml" />
  </group>
  <node name="low_lvl_ctrl" pkg="gripper_controls" type="low_lvl_ctrl" output="screen"/>
  <node name="high_lvl_ctrl" pkg="gripper_controls" type="high_lvl_ctrl" output="screen"/>
</launch>
